crates/pallets/starknet/src/tests/call_contract.rs

```diff
@@ -1,10 +1,11 @@
+use blockifier::abi::abi_utils::selector_from_name;
 use frame_support::{assert_ok, bounded_vec};
 use mp_starknet::execution::types::Felt252Wrapper;
 use mp_starknet::transaction::types::InvokeTransaction;
 use sp_core::ConstU32;
 use sp_runtime::BoundedVec;
 
-use super::constants::TOKEN_CONTRACT_CLASS_HASH;
+use super::constants::{MULTIPLE_EVENT_EMITTING_CONTRACT_ADDRESS, TOKEN_CONTRACT_CLASS_HASH};
```

Comment:
you don't use the other contract?

Reply:
the other contract is initialized in a storage var of this contract as a way to test external event emission